src/Error/SentryHandler.php

```diff
@@ -68,6 +68,10 @@ public function captureMessage($message, $params = [], $data = [], $stack = fals
             return false;
         }
 
+        if (is_callable(Configure::read('CakeMonitor.Sentry.extraDataCallback'))) {
+            $data['extra']['extraDataCallback'] = call_user_func_array(Configure::read('CakeMonitor.Sentry.extraDataCallback'), []);
```

Comment:
Using `call_user_func()` you could omit the empty array if you don't have any parameters